app/views/MediaAtom/defaultEmbed.scala.html

```diff
@@ -0,0 +1,24 @@
+@import com.gu.media.model.{VideoAsset, YouTubeAsset, SelfHostedAsset}
+@(asset: Option[VideoAsset], posterUrl: Option[String])
+
+@defining((asset, posterUrl)) {
+  case (None, Some(poster)) => {
+    <img src="@poster"/>
+  }
+
+  case (Some(YouTubeAsset(id)), _) => {
+    <iframe frameborder="0" allowfullscreen="true" src="https://www.youtube.com/embed/@id"></iframe>
+  }
+
+  case (Some(SelfHostedAsset(sources)), Some(poster)) => {
+    <video controls="controls" poster="@poster">
```

Comment:
can we add `preload="metadata"` so the browser won't fetch the video on page load